packages/core/src/util/renderer.ts

```diff
@@ -80,11 +82,284 @@ import {
 } from '../i18n/arrayTranslations';
 import { resolveSchema } from './resolvers';
 import cloneDeep from 'lodash/cloneDeep';
+import { has } from 'lodash';
+import { all, any } from 'lodash/fp';
```

Comment:
We use the modular imports of lodash, see line 83,84.

Reply:
I see, I have changed it in my latest commit.